Dependency on "events" library not declared.

## Description

lib-jitsi-meet depends on the [events](https://github.com/browserify/events) package. This depencency should be declared in the `package.json`

---

## Current behavior
When packaging the library with [esbuild](https://esbuild.github.io) I get the following errors:
```
$ esbuild app.ts --bundle --target=es2016 --sourcemap --define:__filename='"filename"' --outfile=build/app.js
✘ [ERROR] Could not resolve "events"

    node_modules/@lyno/lib-jitsi-meet/dist/modules/detection/TrackVADEmitter.js:6:41:
      6 │ const events_1 = __importDefault(require("events"));
        ╵                                          ~~~~~~~~

  The package "events" wasn't found on the file system but is built into node. Are you trying to
  bundle for node? You can use "--platform=node" to do that, which will remove this error.

✘ [ERROR] Could not resolve "events"

    node_modules/@lyno/lib-jitsi-meet/dist/JitsiMediaDevices.js:25:41:
      25 │ const events_1 = __importDefault(require("events"));
         ╵                                          ~~~~~~~~

  The package "events" wasn't found on the file system but is built into node. Are you trying to
  bundle for node? You can use "--platform=node" to do that, which will remove this error.

✘ [ERROR] Could not resolve "events"

    node_modules/@lyno/lib-jitsi-meet/dist/modules/statistics/statistics.js:25:41:
      25 │ const events_1 = __importDefault(require("events"));
         ╵                                          ~~~~~~~~

  The package "events" wasn't found on the file system but is built into node. Are you trying to
  bundle for node? You can use "--platform=node" to do that, which will remove this error.

✘ [ERROR] Could not resolve "events"

    node_modules/@lyno/lib-jitsi-meet/dist/modules/statistics/PrecallTest.js:16:41:
      16 │ const events_1 = __importDefault(require("events"));
         ╵                                          ~~~~~~~~

  The package "events" wasn't found on the file system but is built into node. Are you trying to
  bundle for node? You can use "--platform=node" to do that, which will remove this error.

✘ [ERROR] Could not resolve "events"

    node_modules/@lyno/lib-jitsi-meet/dist/JitsiConference.js:35:41:
      35 │ const events_1 = __importDefault(require("events"));
         ╵                                          ~~~~~~~~

  The package "events" wasn't found on the file system but is built into node. Are you trying to
  bundle for node? You can use "--platform=node" to do that, which will remove this error.

✘ [ERROR] Could not resolve "events"

    node_modules/@lyno/lib-jitsi-meet/dist/modules/util/Listenable.js:6:41:
      6 │ const events_1 = __importDefault(require("events"));
        ╵                                          ~~~~~~~~

  The package "events" wasn't found on the file system but is built into node. Are you trying to
  bundle for node? You can use "--platform=node" to do that, which will remove this error.

6 of 13 errors shown (disable the message limit with --log-limit=0)

```

---

## Expected Behavior

Packaging the library with esbuild should not result in errors.

---

## Possible Solution

Proposed solutions:

- add 'events' as a dependency
- add 'events' as a peer depenency

---

## Steps to reproduce

```js
import JitsiMeetJS from '@lyno/lib-jitsi-meet';

JitsiMeetJS.init({});
```

package.json

```json
{
  "name": "jitsi-meet-experiments",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "scripts": {
    "build": "esbuild app.ts --bundle --target=es2016 --sourcemap --define:__filename='\"filename\"' --outfile=build/app.js",
    "serve": "serve ."
  },
  "devDependencies": {
    "esbuild": "^0.14.5",
    "serve": "^13.0.2",
    "typescript": "^4.5.4"
  },
  "dependencies": {
    "@lyno/lib-jitsi-meet": "^2.6726.2",
    "jquery": "^3.6.0"
  }
}
```

---

# Environment details


---

--- a/package.json
+++ b/package.json
@@ -23,6 +23,7 @@
     "async": "3.2.2",
     "base64-js": "1.5.1",
     "current-executing-script": "0.1.3",
+    "events": "3.3.0",
     "jquery": "3.6.0",
     "lodash.clonedeep": "4.5.0",
     "lodash.debounce": "4.0.8",

